From 1530af24085deb5d6798c88147fe7090f0048c72 Mon Sep 17 00:00:00 2001
From: Ben Walton <bwalton@opencsw.org>
Date: Wed, 13 Apr 2011 20:54:02 +0200
Subject: [PATCH] Update AC_INIT use to make configure handle docdir

This patch allows AC_INIT to properly set the PACKAGE variables in the
Makefiles.  In turn, this allows docdir use to work correctly.  With
the new behaviour, things placed in docdir go to share/doc/bacula.
Previously the would end up in share/doc.

We move the detection of the source directory to a separate
AC_CONFIG_SRCDIR macro call.

Signed-off-by: Ben Walton <bwalton@opencsw.org>
---
 autoconf/configure.in |    6 ++++--
 1 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/autoconf/configure.in b/autoconf/configure.in
index 0ce9d94..dffcd5f 100644
--- a/autoconf/configure.in
+++ b/autoconf/configure.in
@@ -3,8 +3,10 @@ dnl
 dnl Process this file with autoconf to produce a configure script.
 dnl
 dnl require a recent autoconf
-AC_PREREQ(2.61)
-AC_INIT(src/version.h)
+AC_PREREQ(2.62)
+AC_CONFIG_SRCDIR(src/version.h)
+AC_INIT([bacula], my_esyscmd([sed -n -e 's/^.*VERSION.*"\(.*\)"$/\1/p' ${srcdir}/src/version.h]))
+
 BUILD_DIR=`pwd`
 cd ..
 TOP_DIR=`pwd`
-- 
1.7.3.2

